### PR TITLE
[SPARK-15194] [ML] Add Python ML API for MultivariateGaussian

### PR DESCRIPTION
**Pull request by praveendareddy21**
## What changes were proposed in this pull request?

Added Multivariate Gaussian and tests to match Scala's ML API.

Ran pep8 and other doc changes.
Reopening Pull request from 2.0 branch on admin's request.
## How was this patch tested?

Unit tests : MultiVariateGaussianTests

Also tested manually on local setup.

### CODE REVIEW

**Comment by holdenk** [comment]
Is this something you are still working on?

**Comment by praveendareddy21** [comment]
@holdenk 
The work on this feature is done. This PR is reopen of PR on 2.0 branch.
refer https://github.com/apache/spark/pull/13248
I waiting for admins to verify and merge.

**Comment by holdenk** [comment]
Ok so you are blocked on reviewers, but still active on the PR then yes? You might want to try pinging (by using @[username] some of the reviewers from your previous PR https://github.com/apache/spark/pull/13248 )

File: python/pyspark/ml/stat/__init__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (holdenk) [inline python/pyspark/ml/stat/__init__.py]: Super minor but probably don't need this blank line.

**Comment by praveendareddy21** [comment]
@MechCoder 
can you review and merge this PR?
refer https://github.com/apache/spark/pull/13248 for discussions.

**Pull request closed without merging**